src/AclExtras.php

```diff
@@ -178,10 +178,10 @@ protected function _updateControllers($root, $controllers, $plugin = null)
         // look at each controller
         $controllersNames = [];
         foreach ($controllers as $controller) {
-            $tmp = explode('/', $controller);
+            $tmp = explode(DS, $controller);
             $controllerName = str_replace('Controller.php', '', array_pop($tmp));
             $controllersNames[] = $controllerName;
-            $path = $this->rootNode . '/' . $pluginPath . $controllerName;
+            $path = $this->rootNode . DS . $pluginPath . $controllerName;
```

Comment:
I agree with @ceeram . Not necessarily needed. Both slashes work fine and would make the DB cross OS save if it stayed.